cmd/ncproxy/ncproxy.go

```diff
@@ -84,7 +84,6 @@ func (s *grpcService) AddNIC(ctx context.Context, req *ncproxygrpc.AddNICRequest
 	}
 	if agent, ok := s.containerIDToComputeAgent.get(req.ContainerID); ok {
 		caReq := &computeagent.AddNICInternalRequest{
-			ContainerID:  req.ContainerID,
```

Comment:
My ncproxy cache is getting purged, we don't need to pass this along anymore (if we even needed to at all)? I don't believe it was needed but we don't log it just for tracing purposes or anything in the shim?

Reply:
I don't know of a specific reason to. If it was for logging, we can just log the compute agent's utility VM in compute agent.

Reply:
We added an annotation so they could specify a custom "container id" to use to pass during requests https://github.com/microsoft/hcsshim/blob/master/cmd/containerd-shim-runhcs-v1/pod.go#L178-L181, so the UVM ID might not be the same

Reply:
Ah, good point. I can add it back just for future tracing needs.

Reply:
added back

Reply:
I still see them getting deleted here, although I see that you added back where they get logged (but they'll just be empty strings as they're not getting passed here)

Reply:
Whoops, added back.